### PR TITLE
[KYUUBI #3219] Error renew delegation tokens: Unknown version of delegation token 8

### PR DESCRIPTION
**Pull request by zhouyifan279**
### _Why are the changes needed?_
Spark SQL Engine failed to get issue date from some DelegationTokenIdentifiers, such as OzoneTokenIdentifier.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by zhouyifan279** [comment]
cc @yaooqinn @MLikeWater

**Comment by zhouyifan279** [comment]
Also cc @pan3793

**Review by yaooqinn — APPROVED** [review]

File: kyuubi-common/src/main/scala/org/apache/kyuubi/util/KyuubiHadoopUtils.scala [hunk 1]
```diff
@@ -85,13 +87,23 @@ object KyuubiHadoopUtils {
...
+        val in = new DataInputStream(buf)
+        Try(tokenIdentifier.readFields(in)) match {
+          case Success(_) =>
+            Some(tokenIdentifier.getIssueDate)
+          case Failure(_) =>
+            None
+        }
+      case _ =>
+        None
+    }
```
Comment (pan3793) [inline kyuubi-common/src/main/scala/org/apache/kyuubi/util/KyuubiHadoopUtils.scala:107]: For debug purposes, I suggest add some clues in negative case

**Review by pan3793 — APPROVED** [review]
LGTM, only one comment

File: kyuubi-common/src/main/scala/org/apache/kyuubi/util/KyuubiHadoopUtils.scala (hunk 1 above)
Comment (zhouyifan279) [reply kyuubi-common/src/main/scala/org/apache/kyuubi/util/KyuubiHadoopUtils.scala:110]: Done

**Comment by pan3793** [comment]
Thanks, merging to master

**Pull request closed without merging**